internal/auth/authz/aap.go

```diff
@@ -0,0 +1,131 @@
+package authz
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+	"slices"
+
+	"github.com/flightctl/flightctl/internal/auth/common"
+	"github.com/fsnotify/fsnotify"
+	"github.com/sirupsen/logrus"
+	"sigs.k8s.io/yaml"
+)
+
+const (
+	AAP_PERMISSIONS_CONFIG_FILE = "/etc/permissions/config.yaml"
```

Comment:
Should we keep config files together?
```suggestion
	AAP_PERMISSIONS_CONFIG_FILE = "/etc/flightctl/permissions.yaml"
```

Reply:
+100, updated